src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/RSASignatureFormatter.cs

```diff
@@ -0,0 +1,99 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.Cryptography.Tests;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Rsa.Tests
+{
+    public partial class RSASignatureFormatterTests : AsymmetricSignatureFormatterTests
+    {
+        [Fact]
+        public static void VerifySignature_SHA1()
+        {
+            using (RSA rsa = RSAFactory.Create())
+            {
+                var formatter = new RSAPKCS1SignatureFormatter(rsa);
+                var deformatter = new RSAPKCS1SignatureDeformatter(rsa);
+                VerifySignature(formatter, deformatter, SHA1.Create(), "SHA1");
+            }
+        }
+
+        [Fact]
+        public static void VerifySignature_SHA256()
+        {
+            using (RSA rsa = RSAFactory.Create())
+            {
+                var formatter = new RSAPKCS1SignatureFormatter(rsa);
+                var deformatter = new RSAPKCS1SignatureDeformatter(rsa);
+                VerifySignature(formatter, deformatter, SHA256.Create(), "SHA256");
+            }
+        }
+
+        [Fact]
+        [ActiveIssue(0)] // todo: add issue. This would work on desktop clr because it uses CryptoConfig which is not case sensitive.
```

Comment:
We need to determine what semantics we want with casing. Oid is case-sensitive. We can accept that we will fail here and be different than desktop, but the issue is that we throw a different exception (WindowsCryptographicException : The requested operation is not supported) vs when a completely bogus hash is used (with throwsCryptographicUnexpectedOperationException)